### ISSUE TITLE
Higher difficulty enchancements : Ideas and suggestions 

### ISSUE THREAD

**Issue by theengineertcr** [eezstreet/SWATEliteForce#109]
Post what you wish to see in higher difficulties to make the game much more difficult than it is now.

Lone wolf:No officers will be with you to assist.

No mistakes:Failing a mission is an instant campaign failure.

Easy and Normal Difficulty:No difference

Hard difficulty:Most Gui elements are hidden except for crosshair and command menu, more aggressive suspects(higher morale, increased chance of picking up weapons and attacking and fleeing if no officer is in view after surrendering), Suspects can escape from missions if they reach an exit point, Civilians can escape from missions if they reach an exit point, Civilains will flee if they are left alone.

Elite Difficulty:All Gui elements except for command menu is hidden, Suspects after taking cover can throw a flashbang towards an area or doorway where an Officer is spotted(restricted to some specific archetypes), Armored suspects will have less stinger duration, when a suspect is close to an officer, they will attempt to punch them, some specific suspects cannot be tased(resisting) and will not comply unless if they've been unarmed with no suspects nearby to help them(mercenaries in hospital and some specific archetypes), mercenary suspects and some specific archeypes will be mostly quiet unless if they call for help.

**Comment by jose21crisis** [comment]
Changes I'm against:
Lone Wolf: makes no sense from a realistic point of view

All Gui elements except for command menu is hidden: No, until we have iron sights.

Suspects can escape from missions if they reach an exit point: Why not making this behavior base to start with, instead of difficulty based?

Civilains will flee if they are left alone.: Make this standard behavior too.

Suspects after taking cover can throw a flashbang towards an area or doorway where an Officer is spotted: NO........the game wasn't designed to allow suspects to have grenades for a reason.....and if we do give them grenades they have to be frags, no flashbangs. Otherwise, just use the tac helmet.

Armored suspects will have less stinger duration: This should be on all difficulties and game modes, not just Elite.

some specific suspects cannot be tased(resisting): Unless we give some suspects heavy armor and make heavy armor users be immune to tasers, no.

and will not comply unless if they've been unarmed with no suspects nearby to help them: This change would mean that if you enter a room with two suspects, and you use a lot of LTL weapons on them, they won't comply. That would mean that LTL weapons would be useless in missions with those specific suspects. And would also mean that you have to kill the suspects. Not the point of the game.

**Comment by theengineertcr** [comment]
Alright, then time to get cracking: Why until we have ironsights? People should be able to know that the center of the screen is where you have to aim, and that you have to be slow otherwise you are inaccurate.

Lone wolf:When you take permanent AI death(a feature we already have) and the fact that the base game allows you to make custom based missions where you can be lone wolf, I don't see this as an issue. 

Heavy armour:Mercenary Suspects already have heavy armour(Hospital suspects), they are completely bullet sponges and take time to kill if you are unlucky. I do believe we can enchance this by giving suspects heavy armour. 

Suspects carrying frag grenades/Game isn't designed to be this way:I disagree. Suspects feel lame, weak and dumb. Making them actually violent, tactical and smart is something I'd like to see, mostly limiting it to mercenaries. Not all of them should just flee and then hide somewhere, where then you can just open the door, throw a nade, and get them to comply. They should be able to use flashbangs and or frags, I don't mind as long as it makes sense in the scenario.  Not all SWAT members use a helmet, and in singleplayer, you're not supposed to look at the flashbang after throwing them because they will and can still blind you. I believe that looking away at a flash should be mandatory while using the tactical helmet as to not blind yourself. It would make much more sense that way. 

Civilians and Suspects fleeing when left alone:I don't mind these being default behaviour, but on higher difficulties they will have a higher chance to, even if low on morale. 

All suspects will not comply unless if they've been unarmed:Yea, I understand. I mainly mean this for mercenary suspects. The ones in hospital don't feel like mercenaries and I wanted them to use SWAT 3 behaviour and voices they sound much more better. The general suspects(victory imports and what not) will not do this. So let me rephrase this idea:Depending on their archetype, suspects will never comply unless if their morale is completely drained and if another suspect with high morale is covering them.

Suspects will flee to exit point to be base behaviour:I mainly wanted this to be restricted to higher difficulty but if this is to be base behaviour, higher difficulties will make this a main priority of suspects in certain missions.

I'd like to hear other ideas from people, Jose, you got any?

**Comment by kevinfoley** [comment]
The main concern I have is that there is no penalty for shooting a suspect or hostage as long as they aren't incapacitated or killed. If you equip a weak weapon like the MP5SD with AP rounds, you can shoot a hostage in the leg half a dozen times with no penalty. Likewise, you can shoot a suspect who doesn't have their weapon raised or has already dropped their weapon.

Obviously, in real life cops don't get away with shooting unarmed individuals just because the wound isn't life-threatening. I think a penalty should be applied immediately for shooting any character that isn't an immediate threat, even if they aren't incapacitated or killed by the shot. However, there should be a grace period of ~.5 seconds for shooting a suspect after he starts surrendering. I've gotten penalized too many times for downing a suspect who drops his weapon in the middle of the burst I'm firing.

**Comment by jose21crisis** [comment]
I would make a new penalty: Unauthorized use of Less Lethal Force. If the target is a suspect, is not a threat or is not issued complaince, you get a penalty. If the target is a civilian, the weapon used is not Less-Lethal and the user has not issued compliance, you get a penalty.

**Comment by jose21crisis** [comment]
Also, make it so that the game diferentiates "Hostages" from "Civilians (aka Innocent Bystanders)". In game all non-hostlies are "civilians". I would make it so that if a "civilian" dies, the mission doesn't inmediatly fail, unless the killer is a SWAT operator. However if a "Hostage" dies, the mission fails if the hostage is a named person, or give a more serious penalty if it is not named. For example, in Northside Vending, is Jeremy Walsh (the downed guy you need to rescue) dies, the mission would fail, but if any of the civilians in the rest of the area dies, the mission doesn't fail.

**Comment by theengineertcr** [comment]
More high difficulty ideas are coming. To differentiate between SWAT 4 gameplay and new gameplay, hard difficulty and above will feature new stuff, normal difficulty will not have ANY medium-large changes, only minor ones, and an extremely small chance to do something on higher difficulties(recommend lowering the new stuff that have been already added to a minimum of 5% to occur during normal difficulty, e.g moving and shooting and etc).
